Kickstarter-iOS/Views/Cells/PledgePaymentMethodAddCell.swift

```diff
@@ -0,0 +1,76 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class PledgePaymentMethodAddCell: UITableViewCell, ValueCell {
+  // MARK: - Properties
+
+  private lazy var selectionView: UIView = { UIView(frame: .zero) |> \.backgroundColor .~ .ksr_grey_200 }()
```

Comment:
I personally prefer to add the style here too, but in this case, shouldn't we a `selectionViewStyle` for consistency?